docs/source/installation.rst

```diff
@@ -88,13 +88,14 @@ Development version install
 
 Development versions of ``sktime`` are installs from static snapshots of the ``sktime`` repository.
 
-Installing development versions is useful for testing, but not recommended
-for development and contribution. For development and contribution, see the next section
-on full developer setup.
+.. warning::
+    An sktime development version install is not recommended for development and contributions.
```

Comment:
perhaps say clearer, "contributions to `sktime`". Also, the editable installs are development version installs, the problem is that this is non-editable, perhaps clarify that point? "plain develpment version install" or similar?

Development version installs (non-editable) are also useful for development, say, of an E2E pipeline in deployment, so perhaps also worth to be clear about "development of `sktime`" point throughout.

Idea: say at the start for whom the given install workflow is, as the first thing?